Docker.DotNet.sln

```diff
@@ -1,39 +1,93 @@
 ﻿
 Microsoft Visual Studio Solution File, Format Version 12.00
-# Visual Studio 14
-VisualStudioVersion = 14.0.25123.0
+# Visual Studio 15

```

Comment:
Yeah, unfortunately the `MSBuild` tooling require VS15 :( And yes, if you don't want VS to build, you can use command line.